core/src/components/slider/slider.ts

```diff
@@ -519,7 +519,8 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 	// TODO define the intersection value
```

Comment:
you can remove the TODO as normalization was the goal of defining the intersection value